### PR TITLE
ENH: Change needed for SAGE2

### PR DESCRIPTION
**Pull request by ElDeveloper**
Allow users to initialize a controller with a webgl canvas, this is needed to instantiate the controller in the SAGE2 environment.

### CODE REVIEW

File: emperor/support_files/js/controller.js [hunk 1]
```diff
@@ -21,9 +21,12 @@ define([
    * @param {dm} a DecompositionModel object that will be
    * represented on screen.
    * @param {divid} the jQuery id correponding to the controller
+   * @param {webglcanvas} the canvas to use to render the information, this
```
Comment (squirrelo) [inline emperor/support_files/js/controller.js]: information, this -> information. This

**Comment by squirrelo** [comment]
one comment, then 👍

**Comment by ElDeveloper** [comment]
@squirrelo comment addressed.

**Comment by squirrelo** [comment]
I don't have merge privileges, but 👍

**Pull request merged**